### PR TITLE
lighthouse-scores-stats element

### PR DESCRIPTION
**Pull request by samthor**
So previously the `lighthouse-scores` element _contained_ the `lighthouse-scores-audits` element (but, it also actually _ran_ Lighthouse for us).

I've kept them separate for now, although this `-stats` element contains both the sparklines and the `-audits`. I guess there's three choices here:

0) keep as-is
1) merge `-audits` into this element (since they're all about showing info from a single Lighthouse run)
2) compose `-audits` inside this element (like v1, but without the controller logic)
3) split this element into `-stats` and `-metrics` (sparklines vs. metrics)—since that way the three categories of things we show are all their own element

Screen Shot 2019-07-29 at 15 21 13

(Missing working sparklines as #1216 is still out for review)

### CODE REVIEW

File: src/lib/components/LighthouseScoresStats/_styles.scss [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+    &.lh-score--average::before {
+      background-color: $AVERAGE_COLOR;
+    }
+
+    &.lh-score--pass::before {
+      background-color: $PASS_COLOR;
+    }
+  }
+
+  @media screen and (min-width: $BREAKPOINT_VALUE_DESKTOP) {
```
Comment (robdodson) [inline src/lib/components/LighthouseScoresStats/_styles.scss]: Mind adding deprecations comment as a block before all of these media queries?

**Comment by robdodson** [comment]
> split this element into -stats and -metrics (sparklines vs. metrics)—since that way the three categories of things we show are all their own element

I vote for this option. I feel like a lot of the elements do too much work so they're tough to review. I'd rather a parent grab the LH report and feed it to stats and metrics children.

**Comment by samthor** [comment]
It's starting to look like it works again.

Screen Shot 2019-08-01 at 15 55 27

I've added `web-profile` which does very little right now, but adds some demo data. I'll work on wiring everything up now.

**Comment by robdodson** [comment]
btw I think if you remove `start` from the end of the `npm run build` command then the tests will start passing.

File: src/lib/components/Profile/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview An element which wraps behavior of the Profile.
```
[the remark this answers is not in the document]
Comment (samthor) [reply src/lib/components/Profile/index.js]: Done

**Comment by samthor** [comment]
I've removed the demo data and renamed the element. So if you're happy I suggest squash merging this (to avoid the ~300kb of data).

**Pull request merged**